### PR TITLE
Feature 12474: Add License step, Tag and Object License to WizardBundle

### PR DESCRIPTION
**Pull request by mreyk**
- Add aditional configurable step of accepting a license text.
- In MultimediaObject metadata step: 
  * Add field of Tag. Optional.
  * Add field of License. Optional.

### CODE REVIEW

File: src/Pumukit/WizardBundle/Controller/DefaultController.php [hunk 1]
```diff
@@ -21,12 +22,44 @@ class DefaultController extends Controller
...
+
+    /**
+     * @Template()
+     */
     public function seriesAction(Request $request)
     {
-        $formData = $request->get('pumukitwizard_form_data');
+        $formData = $request->get('pumukitwizard_form_data', array());
+        $licenseService = $this->get('pumukit_wizard.license');
+        $response = $licenseService->isLicenseEnabledAndAccepted($formData, $request->getLocale());
```
Comment (rubenrua) [inline src/Pumukit/WizardBundle/Controller/DefaultController.php]: Return a `Response` in a service is not a good pattern. (And yes, we used in the past)

File: src/Pumukit/WizardBundle/Services/LicenseService.php [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+    /**
+     * Render license not accepted.
+     *
+     * @param array  $formData
+     * @param string $locale
+     *
+     * @return Response
+     */
+    private function renderLicenseNotAccepted($formData = array(), $locale = null)
```
Comment (rubenrua) [inline src/Pumukit/WizardBundle/Services/LicenseService.php]: Better in a controller

File: src/Pumukit/WizardBundle/Controller/DefaultController.php [hunk 3]
```diff
@@ -21,12 +22,44 @@ class DefaultController extends Controller
     /**
      * @Template()
      */
+    public function licenseAction(Request $request)
+    {
+        $formData = $request->get('pumukitwizard_form_data', array());
+        $licenseService = $this->get('pumukit_wizard.license');
+        if (!$licenseService->isEnabled()) {
+            if (array_key_exists('series', $formData)) {
```
Comment (mreyk) [inline src/Pumukit/WizardBundle/Controller/DefaultController.php]: Fix in `15dc0b2b79e14bb0b587ac95b57e1f9515ba968a`

File: src/Pumukit/WizardBundle/Controller/DefaultController.php (hunk 1 above)
Comment (mreyk) [inline src/Pumukit/WizardBundle/Controller/DefaultController.php]: Fix in commit bb653ac4bb58d6fd887a2bd31a5cc32661777ca1

File: src/Pumukit/WizardBundle/Services/LicenseService.php (hunk 2 above)
Comment (mreyk) [inline src/Pumukit/WizardBundle/Services/LicenseService.php]: Fix in commit bb653ac4bb58d6fd887a2bd31a5cc32661777ca1

**Comment by rubenrua** [comment]
See #261

**Pull request closed without merging**